wp-content/themes/core/pcss/base/typography/_anchors.pcss

```diff
@@ -28,8 +28,8 @@ a {
 
 .anchor,
 .t-content a {
-	border-bottom: 2px solid var(--color-primary);
 	color: var(--color-link);
+	font-weight: var(--font-weight-semibold);
```

Comment:
Note that when you handle global project anchor styles, you'll typically find you need to do a "reset" pass in situations where anchors might be used (image with an anchor, youtube/vimeo lazy implementation, etc.) ; )

Reply:
This was my misunderstanding of the inline link style. Removed it!